test/test-applications/integrations/Samples.Owin.WebApi2/Properties/launchSettings.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
We can use this file in .NET Framework apps? Is it an OWIN thing?

Reply:
🤷 😄 I assume it got created by accident, but given it has all the vars, I'm guessing it works? 🤔  Haven't tried it but I guess @zacharycmontoya did!

Reply:
I wrote the OWIN application as an SDK app, so I think it works! An easy way to verify this is to delete it and see what happens 😄

Reply:
Confirmed it works. Thanks!